_posts/2020-11-11-cockpit-232.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Cockpit 232
+author: kkoukiou
+date: 2020-11-11
+tags: cockpit linux
+slug: cockpit-232
+category: release
+summary: Cockpit with Dashboard and Machines improvements
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/).
+We release regularly. Here are the release notes from Cockpit version 232.
+
+### Improved editing hosts in the switcher
+
+When editing the parameters of a host in the Shell or the Dashboard pages,
+Cockpit no longer tries to change the name of the remote host.
+Instead, it will connect to a different host when you change the address in the "Edit host" dialog.
+
+![edit host new](/images/edit-host-new.png){:.screenshot}
+
+### Machines: Reimplement the design of the error notifications
+
+Error notifications for  virtual machines, storage pools and virtual networks are now displayed
```

Comment:
double space "for _ _ virtual"

Also, please use Oxford comma: "machines, pools*,* and virtual networks"

Reply:
done